### PR TITLE
chore: Remove trailing slash from anchor url

### PR DESCRIPTION
**Pull request by ukstv**

### CODE REVIEW

File: packages/core/src/ceramic.ts [hunk 1]
```diff
@@ -377,7 +377,7 @@ export class Ceramic implements CeramicApi {
     let anchorService = null
     if (!config.gateway) {
       const anchorServiceUrl =
-        config.anchorServiceUrl || DEFAULT_ANCHOR_SERVICE_URLS[networkOptions.name]
+        config.anchorServiceUrl.replace(/\/$/, '') || DEFAULT_ANCHOR_SERVICE_URLS[networkOptions.name]
```
Comment (stbrody) [inline packages/core/src/ceramic.ts:380]: can you add a comment what the replace call is doing?  I find regexes incredibly opaque and difficult to read

Comment (v-stickykeys) [reply packages/core/src/ceramic.ts:380]: Would be even better to store the regex in a variable with a helpful name. So no need for comment

Comment (ukstv) [reply packages/core/src/ceramic.ts]: @stbrody @v-stickykeys Extracted variable.

**Review by stbrody — APPROVED** [review]
lgtm, thanks!

**Pull request merged**